app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -113,6 +137,7 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         // as LP stores it's state internally and check against it
         LeanplumInternal.setCalledStart(false)
         LeanplumInternal.setHasStarted(false)
+        leanplumJob!!.cancel()
```

Comment:
This needs to be `leanplumJob?.cancel()`. If there's ever overlap of starting/stopping we don't want crash with an NPE, but simply do nothing as there is no job to cancel.